core/controller/src/main/scala/whisk/core/entitlement/Entitlement.scala

```diff
@@ -77,16 +76,14 @@ protected[core] object EntitlementProvider {
  * A trait that implements entitlements to resources. It performs checks for CRUD and Acivation requests.
  * This is where enforcement of activation quotas takes place, in additional to basic authorization.
  */
-protected[core] abstract class EntitlementProvider(config: WhiskConfig, loadBalancer: LoadBalancer)(
+protected[core] abstract class EntitlementProvider(config: WhiskConfig, loadBalancer: LoadBalancer, controllerInstance: Int)(
```

Comment:
Related to my comment above - the leaking of the instance into the entitlement strikes me as wrong. I understand why it's happening but suggests we're not quite done on this front. Redis perhaps will allow us to address the inconsistency.